api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -49,25 +59,32 @@ public void test_deleteInstitution() {
 
   @Test
   public void test_getInstitutions() {
-    assertThat(service.getInstitutions()).containsExactlyElementsIn(ImmutableList.of(testInst));
+    assertThat(service.getInstitutions()).containsExactly(testInstAfterRT);
 
     final Institution otherInst =
-        new Institution().shortName("otherInst").displayName("The Institution of testing");
+        new Institution()
+            .shortName("otherInst")
+            .displayName("The Institution of testing")
+            .emailDomains(new ArrayList<>())
+            .emailAddresses(new ArrayList<>());
     service.createInstitution(otherInst);
-    assertThat(service.getInstitutions())
-        .containsExactlyElementsIn(ImmutableList.of(testInst, otherInst));
+    assertThat(service.getInstitutions()).containsExactly(testInstAfterRT, otherInst);
 
     service.deleteInstitution(testInst.getShortName());
-    assertThat(service.getInstitutions()).containsExactlyElementsIn(ImmutableList.of(otherInst));
+    assertThat(service.getInstitutions()).containsExactly(otherInst);
   }
 
   @Test
   public void test_getInstitution() {
-    assertThat(service.getInstitution(testInst.getShortName())).hasValue(testInst);
+    assertThat(service.getInstitution(testInst.getShortName())).hasValue(testInstAfterRT);
     assertThat(service.getInstitution("otherInst")).isEmpty();
 
     final Institution otherInst =
-        new Institution().shortName("otherInst").displayName("The Institution of testing");
+        new Institution()
+            .shortName("otherInst")
+            .displayName("The Institution of testing")
+            .emailAddresses(new ArrayList<>())
+            .emailDomains(new ArrayList<>());
```

Comment:
So if you want to use the builder paradigm, you could make a `billed()` method that fills in missing array values and other defaultable things.